### PR TITLE
Adding new features

### PR DESCRIPTION
**Pull request by MargotLepizzera**
- Assertion on TLS version 
- Proxy support on browser tests
- Adding links to MFA form to guide about auth and main browser test page

### Preview

https://docs-staging.datadoghq.com/margot.lepizzera/syn_new_features/synthetics/api_tests/?tab=ssltest#assertions
https://docs-staging.datadoghq.com/margot.lepizzera/syn_new_features/synthetics/browser_tests/#test-configuration
https://docs-staging.datadoghq.com/margot.lepizzera/syn_new_features/synthetics/guide/app-that-requires-login/
https://docs-staging.datadoghq.com/margot.lepizzera/syn_new_features/synthetics/guide/app-that-requires-login/#multi-factor-authentication
https://docs-staging.datadoghq.com/margot.lepizzera/syn_new_features/synthetics/browser_tests/#overview

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/browser_tests/_index.md [hunk 1]
```diff
@@ -24,15 +24,18 @@ further_reading:
 
 Browser tests are scenarios executed by Datadog on your web applications. They run at configurable periodic intervals from multiple locations around the world, from multiple browsers, and devices. These tests verify both that your applications are up and responding to requests, and that any conditions defined in your scenarios are met.
 
+<div class="alert alert-info"> Interested in testing applications that sit behind MFA? Give <a href="/synthetics/guide/app-that-requires-login/#multi-factor-authentication" target="_blank">our dedicated guide </a> a read and fill that <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjx8PDZ8kJ3MD2ehouTri9z_Fh7PoK90J8arRQgt7QFgFxog/viewform?usp=sf_link">form</a> to help us work on the systems that matter the most to your teams.</div>
```
Comment (kayayarai) [inline content/en/synthetics/browser_tests/_index.md:27]:
```suggestion
<div class="alert alert-info">Are you interested in testing applications that sit behind MFA? Read <a href="/synthetics/guide/app-that-requires-login/#multi-factor-authentication" target="_blank">our dedicated guide </a> and <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjx8PDZ8kJ3MD2ehouTri9z_Fh7PoK90J8arRQgt7QFgFxog/viewform?usp=sf_link">send us feedback</a> to help us work on the systems that matter the most to your teams.</div>
```

File: content/en/synthetics/guide/app-that-requires-login.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ You might need to monitor journeys located behind a login. There are two ways to
 
 You can also ensure your credentials are securely stored and obfuscated across the application [using secured global variables](#account-security).
 
+<div class="alert alert-info"> Interested in testing applications sitting behind MFA? Visit the <a href="/synthetics/guide/app-that-requires-login/#multi-factor-authentication" target="_blank">below section</a> and fill that <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjx8PDZ8kJ3MD2ehouTri9z_Fh7PoK90J8arRQgt7QFgFxog/viewform?usp=sf_link">form</a> to help us work on the systems that matter the most to your teams.</div>
```
Comment (kayayarai) [inline content/en/synthetics/guide/app-that-requires-login.md:24]: same suggestions

File: content/en/synthetics/guide/app-that-requires-login.md [hunk 3]
```diff
@@ -48,13 +50,13 @@ An alternative would be to use a non-SSO approach and leverage a regular usernam
...
 
 If your MFA process involves steps performed outside of the browser, such as voice, text message, or opening a mobile application, also consider reaching out to your MFA provider to ask if your MFA settings could be modified or if MFA could be turned off when [identifying requests as coming from Synthetic browser tests][3] (for example, for a specific set of credentials, Synthetic tests specific headers, etc.) for testing purposes.
 Depending on the type of MFA leveraged by your application, [JavaScript steps][4] could help to work around that.
 
+<div class="alert alert-info">We are constantly adding features to help you record test scenarios more easily. Help us work on the MFA systems that matter the most to you by filling that <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjx8PDZ8kJ3MD2ehouTri9z_Fh7PoK90J8arRQgt7QFgFxog/viewform?usp=sf_link">form</a>.</div>
```
Comment (kayayarai) [inline content/en/synthetics/guide/app-that-requires-login.md:58]:
```suggestion
<div class="alert alert-info">We are constantly adding features to help you record test scenarios more easily. Help us work on the MFA systems that matter the most to you by <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjx8PDZ8kJ3MD2ehouTri9z_Fh7PoK90J8arRQgt7QFgFxog/viewform?usp=sf_link">sending us feedback</a>.</div>
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Some wording suggestions. Also, I'll look into the bulleted list problem

File: content/en/synthetics/guide/app-that-requires-login.md (hunk 2 above)
Comment (MargotLepizzera) [reply content/en/synthetics/guide/app-that-requires-login.md]: Done!

**Review by kayayarai — APPROVED** [review]

**Pull request merged**